src/main.ts

```diff
@@ -185,7 +185,7 @@ export async function runAction(): Promise<any> {
   }
 }
 
-export function getArtifactName(config: Config): string {
+export function getArtifactName(config: Config, executionGraphIDTruncate: string): string {
```

Comment:
Looks good I think it is going to be better if you pass the whole `executionGraphId` and then do the truncating inside this method. 

That way, you can change the algorithm much easier later and you don't have to truncate all those tests/invocations. This will also let you add a new unit test that validates that when passing a `config` and an `executionGraphId` the name will have the 8-chars hash appended.

Reply:
Right, I update with the change

Reply:
Tests pass but I think you need to change those tests removing the call to slice when you invoke the getArtifactName function. 

e.g. from 
```
const artifactName = await getArtifactName(config, executionGraphId.slice(0, 8))
```
to
```
const artifactName = await getArtifactName(config, executionGraphId)
```

Reply:
Updated!